Sorry for the delayed response. This fails to build when rebased on top of current `master`:

```
Ranlib Library scene/libscene.linuxbsd.editor.x86_64.a ...
In file included from editor/scu/scu_editor_4.gen.cpp:2:
./editor/editor_node.cpp: In member function 'void EditorNode::_menu_option_confirm(int, bool)':
./editor/editor_node.cpp:3049:57: error: 'RELOAD_CURRENT_PROJECT' was not declared in this scope; did you mean 'PROJECT_RELOAD_CURRENT_PROJECT'?
 3049 |                                         if (p_option == RELOAD_CURRENT_PROJECT) {
      |                                                         ^~~~~~~~~~~~~~~~~~~~~~
      |                                                         PROJECT_RELOAD_CURRENT_PROJECT
```

Can you look into rebasing and fixing the build?

--- a/editor/editor_node.cpp
+++ b/editor/editor_node.cpp
@@ -3045,6 +3045,19 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			}
 
 			if (!p_confirmed) {
+				if (project_run_bar->is_playing()) {
+					if (p_option == PROJECT_RELOAD_CURRENT_PROJECT) {
+						confirmation->set_text(TTR("Stop running project before reloading the current project?"));
+						confirmation->set_ok_button_text(TTR("Stop & Reload"));
+					} else {
+						confirmation->set_text(TTR("Stop running project before exiting the editor?"));
+						confirmation->set_ok_button_text(TTR("Stop & Quit"));
+					}
+					confirmation->reset_size();
+					confirmation->popup_centered();
+					break;
+				}
+
 				bool save_each = EDITOR_GET("interface/editor/save_each_scene_on_quit");
 				if (_next_unsaved_scene(!save_each) == -1) {
 					if (EditorUndoRedoManager::get_singleton()->is_history_unsaved(EditorUndoRedoManager::GLOBAL_HISTORY)) {

